src/platform/graphics/display_configuration.cpp

```diff
@@ -321,10 +359,11 @@ mg::UserDisplayConfigurationOutput::UserDisplayConfigurationOutput(
         subpixel_arrangement(main.subpixel_arrangement),
         gamma(main.gamma),
         gamma_supported(main.gamma_supported),
-        edid(main.edid),
+        edid(main.display_info.raw_edid),
```

Comment:
What is happening with the `edid` members? `DCO::edid` is no longer being set (nor referred to by `UDCO::edid` or the logging, ...)